tumblr-photo-video-ripper.py

```diff
@@ -82,15 +88,16 @@ def _register_regex_match_rules(self):
     def _handle_medium_url(self, medium_type, post):
         try:
             if medium_type == "photo":
-                return post["photo-url"][0]["#text"]
+                return post["url"]
 
             if medium_type == "video":
-                video_player = post["video-player"][1]["#text"]
-                for regex_rule in self.regex_rules:
-                    matched_url = regex_rule(video_player)
-                    if matched_url is not None:
-                        return matched_url
-                else:
+                video_player = post["video_url"]
```

Comment:
Have you referred to api v2 doc. I did not see any `video_url ` attributes for video posts.